src/sentry/models/grouphistory.py

```diff
@@ -149,7 +150,7 @@ class GroupHistoryStatus:
 
 
 class GroupHistoryManager(BaseManager["GroupHistory"]):
-    def filter_to_team(self, team: "Team"):
+    def filter_to_team(self, team: "Team") -> "QuerySet[GroupHistory]":
```

Comment:
if you `from __future__ import annotations` you won't need the quotes on either of these (just adding the import will let `pyupgrade` autofix these too!)

Reply:
Amazing! Thanks for teaching me! 🥔